coral-hive/src/test/java/com/linkedin/coral/hive/hive2rel/HiveToRelConverterTest.java

```diff
@@ -559,6 +559,29 @@ public void testNestedGroupBy() {
     assertEquals(relToHql(rel), expectedSql);
   }
 
+  @Test
+  public void testComment() {
+    final String expected =
+        "LogicalProject(a=[$0], b=[$1], c=[$2])\n" + "  LogicalTableScan(table=[[hive, default, foo]])\n";
+
+    // single-line comments
+    final String sql1 = "--comment0 select\nselect * -- comment1\nfrom foo";
```

Comment:
Could you please use uppercase for keywords similar to the above test cases?

Reply:
Thanks @wmoustafa for the quick review! Updated as requested.